wkcuber/api/Dataset.py

```diff
@@ -307,6 +318,66 @@ def _create_layer(
     ) -> Layer:
         raise NotImplementedError
 
+    def copy_dataset(
+        self, empty_target_ds: "AbstractDataset", args: Namespace = None
```

Comment:
I have added the `Optional` now because it is semantically more correct, but I think mypy automatically understands that parameters with a default value of `None` are optional (at least my IDE understands it)